source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -1142,6 +1155,7 @@
 
 						.udid {
 							word-break: break-all;
+							word-wrap: break-word;
```

Comment:
This way if there is a too long UDID (and on mobile, all of them will be too long), it won't be broken down to multiple lines, since it does not have white space. Is that what we want?

Reply:
The issue appears only in firefox's quantum, there we should add this line, I found this advice somewhere on stackoverflow. :D